client/src/main/java/org/spine3/base/ListStringifier.java

```diff
@@ -0,0 +1,154 @@
+ *
+ *    // The registration of the stringifier.
+ *    final Stringifier<List<Integer>> listStringifer = Stringifiers.listStringifier();
+ *    final Type type = Types.listTypeOf(Integer.class);
+ *    StringifierRegistry.getInstance().register(listStringifier, type);
+ *
+ *    // Obtain already registered stringifier.
+ *    final Stringifier<List<Integer>> listStringifier = StringifierRegistry.getInstance()
+ *                                                                          .getStringifier(type);
+ *
+ *    // Convert to string.
+ *    final List<Integer> listToConvert = newArrayList(1, 2, 3);
+ *
+ *    // The result is: \"1\",\"2\",\"3\".
+ *    final String convertedString = listStringifer.toString(listToConvert);
+ *
+ *    // Convert from string.
+ *    final String stringToConvert = ...
+ *    final List<Integer> convertedList = listStringifier.fromString(stringToConvert);
+ * }
+ *
+ * @param <T> the type of the elements in the list.
+ */
+class ListStringifier<T> extends Stringifier<List<T>> {
+
+    private static final char DEFAULT_ELEMENT_DELIMITER = ',';
+
+    /**
+     * The delimiter for the passed elements in the {@code String} representation,
+     * {@code DEFAULT_ELEMENT_DELIMITER} by default.
+     */
+    private final char delimiter;
+    private final Class<T> listGenericClass;
+    private final String elementDelimiterPattern;
+    private final Escaper escaper;
+
+    /**
+     * That constructor should be used when a custom delimiter is not needed.
+     *
+     * <p>The {@code DEFAULT_ELEMENT_DELIMITER} will be used.
+     *
+     * @param listGenericClass the class of the list elements
+     */
+    ListStringifier(Class<T> listGenericClass) {
+        super();
+        this.listGenericClass = listGenericClass;
+        this.delimiter = DEFAULT_ELEMENT_DELIMITER;
+        escaper = createEscaper(delimiter);
+        elementDelimiterPattern = createElementDelimiterPattern(delimiter);
+    }
+
+    /**
+     * That constructor should be used for providing a custom
+     * delimiter of the elements during conversion.
+     *
+     * @param listGenericClass the class of the list elements
+     * @param delimiter        the delimiter for the passed elements via string
+     */
+    ListStringifier(Class<T> listGenericClass, char delimiter) {
+        super();
+        this.listGenericClass = listGenericClass;
+        this.delimiter = delimiter;
+        escaper = createEscaper(delimiter);
+        elementDelimiterPattern = createElementDelimiterPattern(delimiter);
+    }
+
+    private static String createElementDelimiterPattern(char delimiter) {
+        return Pattern.compile("(?<!\\\\)\\\\\\" + delimiter)
+                      .pattern();
+    }
+
+    @Override
+    protected String toString(List<T> list) {
+        final StringBuilder stringBuilder = new StringBuilder(0);
+        for (T element : list) {
+            final char quote = '"';
```

Comment:
That is fragile. 

The knowledge about quoting the each element is both here and in `org.spine3.base.Stringifiers.isQuotedString`. Making any change to either of these places will lead to a hardly-traceable fail.